ralf/tables/redis_connector.py

```diff
@@ -0,0 +1,41 @@
+import json
+from typing import List, Union
+
+import redis
+
+from ralf.state import Record, Schema
+from ralf.tables.connector import Connector
+
+
+class RedisConnector(Connector):
+    # Not sure how historical would work with Redis so ignoring it for now
+
+    def __init__(self, conn: redis.client.Redis):
+        self.conn = conn
+
+    def add_table(self, schema: Schema, historical: bool):
+        pass
+
+    def update(self, schema: Schema, historical: bool, record: Record):
+        key = getattr(record, schema.primary_key)
+        jsonString = json.dumps(vars(record))
```

Comment:
snake case

Reply:
actually should we pickle instead of json because pickle is nicer to np array and dataframe